src/tests/unit/tests/common/components/cards/how-to-fix-android-card-row.test.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { Mock } from 'typemoq';
+
+import { HowToFixAndroidCardRow, HowToFixAndroidCardRowProps } from 'common/components/cards/how-to-fix-android-card-row';
+import { FixInstructionProcessor } from '../../../../../../injected/fix-instruction-processor';
+
+describe(HowToFixAndroidCardRow, () => {
+    it('renders with matches', () => {
+        const fixInstructionProcessorMock = Mock.ofType(FixInstructionProcessor);
+        const props: HowToFixAndroidCardRowProps = {
+            deps: { fixInstructionProcessor: fixInstructionProcessorMock.object },
+            index: 22,
+            propertyData: {
+                formatAsCode: ["isn't", 'notPresent', 'bold'],
+                howToFix: "This isn't a simple text. It has some words that will be marked as bold.",
+            },
+        };
+
+        const testSubject = shallow(<HowToFixAndroidCardRow {...props} />);
+
+        expect(testSubject.getElement()).toMatchSnapshot();
+    });
+
+    type RenderWithSpaceTestCase = {
+        label: string;
+        formatAsCode?: string[];
+        howToFix: string;
+    };
+
+    const renderWithSpaceTestCases: RenderWithSpaceTestCase[] = [
+        {
+            label: 'when string starts with space',
+            howToFix: ' This is a sample string',
+        },
+        {
+            label: 'when string ends with space',
+            howToFix: 'This is a sample string ',
+        },
+        {
+            label: 'when match is within a word',
+            formatAsCode: ['amp'],
+            howToFix: 'This is a sample string ',
+        },
+        {
+            label: 'when match is on a word',
+            formatAsCode: ['sample'],
+            howToFix: 'This is a sample string ',
+        },
+    ];
+    renderWithSpaceTestCases.forEach(testCase => {
+        it(`renders with whitespace - ${testCase.label}`, () => {
```

Comment:
is it only because you only want `testCase.label` to show up on the test description console output that you are not using `it.each` here?

Reply:
we do have this usage in many places. for eg: content-page.test.tsx